### ISSUE TITLE
Leave original lecture number

### ISSUE THREAD

**Issue by muescha** [FaisalUmair/udemy-downloader-gui#20]
what:
- when i download a file
- i see a lecture
- i like to see the notes from the lecture
- i have section number and lecture number

what i see:
- in each section is count from 1
- but on the website it is counted ongoing

what i expect:
- an option to count ongoing - so that my files in sync with the lection numbers on the website

**Comment by FaisalUmair** [comment]
Ok, for this I wasn't quite sure which way to go because the naming scheme I used was my preferred one. 

Check this #9   

I deliberately left it to the naming scheme it uses currently, we can however change it easily without much effort. But I was waiting someone to point it out so that I know it is something that bothers users.

**Comment by muescha** [comment]
yes - it is the same problem but for an other use case - to be in sync with the original course and read the FAQ there or leave comments in the FAQ

**Comment by muescha** [comment]
i just looked into the iOS App: there is also the ongoing counting. so i think the users are used to have this ongoing counting as default - and via opt out it would be ok to have the "per section counting"

**Comment by FaisalUmair** [comment]
Ok, lets got with the ongoing counting as it sound more natural. 

One thing I noticed on Udemy is that it doesn't detect Quiz as part of counting lectures. But doest count Articles, Presentations etc. Do you think we should also skip quizzes so that they don't interfere with the counting and the lecture structure will match as on Udemy website.

**Comment by muescha** [comment]
definitely then it should match the Udemy website. 

just now i see only max one attachment per lecture. maybe handle a quiz as an attachment to the previous lecture?

**Comment by FaisalUmair** [comment]
Well a Quiz can be in a separate section, so adding it as a attachments to the previous lecture will put the quiz in incorrect section. I think we can simply skip prefix for Quiz just name it as it is. 

And a quiz as attachment is different than quiz being as a lecture. If a lecture has a attachment, we name it as LECTURE_NAME.ATTACHMENT_NUMBER   

And what do you mean by max one attachments per lecture, I think I did not got the point you were making.

**Comment by muescha** [comment]
in my downloaded course i see only one attachment per lecture - not more - maybe this is only in my course that they have only one

**Comment by FaisalUmair** [comment]
Yeah could be so, I tried with multiple attachments and it works.

**Comment by muescha** [comment]
just checked: i found also one lecture with 2 attachments

**Comment by FaisalUmair** [comment]
Can you share the course link? So that I can see what could be wrong. I implemented that attachment download in a way that it downloads all attachments of a course. But it is a very new implementation and I don't expect it to work perfectly and thats why I haven't releases 1.3.0 yet.

**Comment by muescha** [comment]
your implementation is good - there is no error

**Comment by FaisalUmair** [comment]
Ok so for the lecture which has 2 attachments, all attachments were downloaded?

**Comment by muescha** [comment]
i am still stopped at section 14 because of too much downloaded for my ssd :)

**Comment by FaisalUmair** [comment]
No issues, I guess it should work fine. 

Thank you for all your suggestions.

**Comment by muescha** [comment]
at first: i tried the from - to settings - the are not effect if i am on pause. there is no stop button. 
i need to quit Udeler and restart to get effect the `from - to` settings working. after restart it worked.

i done setting: from:244 to:249 
existing lectures: 
```
244+1
245+0
246+0
247+2
248+1
249+1
```
results:
a) 244-247 lectures with all attachements are downloaded
b) 248/249 missing
c)  progress 50% display 3 of 6 - above (per file) ca. 20%  - and left top is a label attachment

**Comment by FaisalUmair** [comment]
When the overall progress stops and only the lecture based progress works it means that an attachment is currently being downloaded. 

And if you mean that the progress is stuck, you can open dev tools and check what it is saying.

**Comment by muescha** [comment]
Next week i can look more into it

**Comment by FaisalUmair** [comment]
Sure, Thanks!
